### ISSUE TITLE
joe still had a bigger z-index 

### ISSUE THREAD

**Issue by longebane** [Nick-Gottschlich/mine-is-bigger#4]
I started my day with a big grin, knowing today was the day that Joe would wipe that smile off his dweeby-ass face. I had something that he didn't. Mine-is-bigger. I finished my latest feature request, and pushed it to production after hours of unit testing and making sure my z index could fill up an entire football stadium.

I clocked out, went home and told my wife and kids they don't need to worry about losing the house anymore. Daddy's got the largest z index around now. 

Come in the next day to see all my things packed up on my desk. Huh? What happened? "Boss wants to see you", Joe said.

So basically, boss said I'm done. That component I've been working on for the last 3 months? Didn't even show up on the website. Was covered up by Joe's sticky header.

(this didn't actually happen. But I wanted to comment on how funny the description was for this package)

**Comment by Nick-Gottschlich** [comment]
Amazing, thank you for this 🙏

**Comment by Nick-Gottschlich** [comment]
![image](https://user-images.githubusercontent.com/10564265/83472195-4c222f80-a44c-11ea-8914-c108e70619e0.png)
